src/sidebarLearn.json

```diff
@@ -186,7 +186,7 @@
         },
         {
           "title": "Removing Effect Dependencies",
-          "path": "/learn/removing-effect-dependencies"              
+          "path": "/learn/removing-effect-dependencies"
```

Comment:
Ideally it should have been fixed by prettier. Lets add JSON to prettier run if it was not there before?

Reply:
yes. I actually noticed this omission while saving the file locally. It's possible that this vulnerability was reported before formatting was put in place.

Reply:
Perhaps you can format it with one click before submitting.